src/online2/online-nnet2-feature-pipeline.cc

```diff
@@ -64,6 +64,17 @@ OnlineNnet2FeaturePipelineInfo::OnlineNnet2FeaturePipelineInfo(
       KALDI_WARN << "--online-pitch-config option has no effect "
                  << "since you did not supply --add-pitch option.";
   }  // else use the defaults.
+  if (config.cmvn_config != "") {
```

Comment:
please clarify here and where you declare these objects, via comments, that this is the CMVN used for the features, not those used inside the ivector extractor (they may be different because we don't include pitch features when estimating the i-vectors).